Source: me#gmail.com
Upstream: rejected
Reason: Install gtk immodule in a better directory.
--- gcin-2.4.2/gtk-im/Makefile.orig	2011-09-22 23:54:27.000000000 +0800
+++ gcin-2.4.2/gtk-im/Makefile	2011-12-03 23:34:42.260589377 +0800
@@ -1,7 +1,7 @@
 include ../config.mak
 
 OBJS = imgcin.o gtkimcontextgcin.o
-GTK2IM=gtk-2.0/immodules
+GTK2IM=gtk-2.0/`pkg-config --variable=gtk_binary_version gtk+-2.0`/immodules
 GTKINC=`pkg-config --cflags gtk+-2.0`
 LDFLAGS=`pkg-config --libs gtk+-2.0`
 IMMODULES=$(libdir)/$(GTK2IM)
